airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -279,6 +279,7 @@ def transform_clickhouse(config: Dict[str, Any]):
             "port": config["port"],
             "schema": config["database"],
             "user": config["username"],
+            "secure": config["ssl"]
```

Comment:
My only concern here is if the config file created by the UI read as boolean and not as a string.

Reply:
Hey, it is boolean:
https://github.com/airbytehq/airbyte/blob/a4e8417ec88d740c3d340b2d1df4f9a69e80117f/airbyte-integrations/connectors/destination-clickhouse/src/main/resources/spec.json#L59-L65

I've managed to run normalization successfully, only I had to change `NORMALIZATION_VERSION = "dev"`, build `SUB_BUILD=PLATFORM ./gradlew build` and test it with UI locally.
https://github.com/airbytehq/airbyte/blob/a4e8417ec88d740c3d340b2d1df4f9a69e80117f/airbyte-workers/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java#L17

However I think it was hard way to test this. 
![](https://habrastorage.org/webt/4c/gt/pu/4cgtpuc6iefiw7l0t1qofkloozi.png)

Reply:
Awesome!